core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -432,26 +432,23 @@ private void checkTempBlockOwnedBySession(long sessionId, long blockId)
    */
   private void abortBlockInternal(long sessionId, long blockId) throws BlockDoesNotExistException,
       BlockAlreadyExistsException, InvalidWorkerStateException, IOException {
-    long lockId = mLockManager.lockBlock(sessionId, blockId, BlockLockType.WRITE);
```

Comment:
also, shall we check in `BlockLockManager` that if a session is applying a write lock while holding any lock is already hold by this session, and fail the second lock application? @aaudiber